client-v2/src/main/java/com/clickhouse/client/api/Client.java

```diff
@@ -444,6 +450,12 @@ public Builder useNewImplementation(boolean useNewImplementation) {
             return this;
         }
 
+        public Builder setHttpCookiesEnabled(boolean enabled) {
+            //TODO: extract to settings string constants
+            this.configuration.put("client.http.cookies_enabled", String.valueOf(enabled));
```

Comment:
What is that used for?

Reply:
- sticky sessions 
- some enterprise proxies like F5 that store routing information in cookies

Reply:
actually there may be opposite case when cookies should not be preserved to avoid sticky session when proxy behavior may not be changed.